gcp_variant_transforms/libs/bigquery_vcf_schema_converter.py

```diff
@@ -325,6 +326,8 @@ def _validate_reserved_field_type(field_schema, reserved_definition):
 
 def _validate_reserved_field_mode(field_schema, reserved_definition):
   schema_mode = field_schema.mode
+  if not schema_mode:
```

Comment:
nit: more pythonic to do `schema_mode = field_schema.mode or bigquery_util.TableFieldConstants.MODE_NULLABLE`